src/test/java/duke/TaskListTest.java

```diff
@@ -0,0 +1,27 @@
+package duke;
+
+import java.util.ArrayList;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class TaskListTest {
+
+    @Test
+    public void addTest() {
+        TaskList tasks = new TaskList();
+        ToDo todo = new ToDo("Buy bread");
+        tasks.add(todo);
+        assertEquals(tasks.size(), 1);
+    }
+
+    @Test
+    public void deleteTest() {
```

Comment:
Similar code quality problem here, verb for command, suggestion: `testDeleteTask`